docs/content/gameobject/readers-writers.md

```diff
@@ -106,8 +107,9 @@ The exact type of these generics depends on the component that the writer is gen
 
 | Field         	| Type               	| Description                	|
 |-------------------|------------------------|--------------------------------|
-| Data  	| TSpatialComponentData              	| Thee data stored inside the component that this Reader is associated with. |
-| Authority | [Authority]({{urlRoot}}/content/glossary#authority) | The [authority](https://docs.improbable.io/reference/latest/shared/concepts/interest-authority#authority) status of the current worker of the component that this Reader is associated with. |
+| EntityId | `EntityId`                 | The EntityId of the underlying SpatialOS entity of this Writer. |
+| Data  	| `TSpatialComponentData`              	| The data stored inside the component that this Reader is associated with. |
+| Authority | [`Authority`]({{urlRoot}}/content/glossary#authority) | The [authority](https://docs.improbable.io/reference/latest/shared/concepts/interest-authority#authority) status of the current worker of the component that this Reader is associated with. |
```

Comment:
```suggestion
| Authority | `Authority` | The authority({{urlRoot}}/content/glossary#authority) status of the current worker of the component that this Reader is associated with. |
```

Probs better to have one backlink to the glossary on the GDK rather than two different glossaries. (Apologies if I messed up the linking before - can't remember.)